planutils/packages/kstar/manifest.json

```diff
@@ -1,6 +1,44 @@
 {
-    "name": "K* planner for the top-k planning problem",
-    "description": "https://github.com/ctpelok77/kstar",
+    "name": "kstar",
+    "description": "Kstar planner",
```

Comment:
Description should include all the info you removed (old name and old description)